### DISCUSSION TITLE
Directus URL that redirects to SSO registration

### DISCUSSION THREAD

**Discussion by mlbiche** [directus/directus#16608] [Archive]
As per now, Directus generates a URL that redirects to SSO login page with proper configuration in query params (response type, code_challenge, client_id, etc.).

Does a similar URL exist that redirects to SSO registration page ?

For example, with a Keycloak SSO provider :

```
[...]/realms/local/protocol/openid-connect/auth
```

becomes

```
[...]/realms/local/protocol/openid-connect/registrations
```

Cheers 🙏

**Comment by aidenfoxx** [comment]
The SSO flows do not support registration endpoints, since registration implementations vary between providers.

**Reply by mlbiche** [reply]
Ok, I understand. Is the code challenge logic reusable within a custom endpoint?
